solr/core/src/test/org/apache/solr/TestSolrCoreMetricProperties.java

```diff
@@ -65,7 +65,7 @@ public void testCoreLevelMetricsEnabled() throws Exception {
   public void testCoreLevelMetricsDisabled() throws Exception {
     System.setProperty("coreLevelMetricsEnabled", "false");
```

Comment:
Tried to debug a little bit on this and it seems to not really test on how system property affect such flag? As I was expecting that this would trigger logic in https://github.com/fullstorydev/lucene-solr/blob/justinsweeney/node-level-metrics-improvements/solr/core/src/java/org/apache/solr/core/SolrXmlConfig.java#L552 but instead, it is the `TestHarness` that is reading such property? https://github.com/fullstorydev/lucene-solr/commit/ebfa0af8a46d4eae102119866a4bea0347f75eaa#diff-de64d167e641f5988140d8c4d6a977beaaec9e2a8144123153f0e126722843dcR209